### ISSUE TITLE
cheatengine not installed. The package was not found with the source(s) listed.

### ISSUE THREAD

**Issue by T00mm** [GameHackingAcademy/vmsetup#1]
Seems like choco doesn't have Cheat Engine anymore in its repositories?

```
cheatengine not installed. The package was not found with the source(s) listed.
 Source(s): 'https://chocolatey.org/api/v2/'
 NOTE: When you specify explicit sources, it overrides default sources.
If the package version is a prerelease and you didn't specify `--pre`,
 the package may not be found.
Please see https://docs.chocolatey.org/en-us/troubleshooting for more
 assistance.
```

**Comment by attilathedud** [comment]
Thanks for letting me know about this! Another person reached out telling me something similar, so I will investigate another method for the future. In the meantime, you can download the regular Cheat Engine installer from their main site.

**Comment by T00mm** [comment]
Yes, thanks for the tutorial, I did indeed just go to the cheat Engine website.

For anyone reading, watch out for the payed advertisement in the installer of Cheat Engine installer. You can skip it safely but do not just enter through it!

**Comment by attilathedud** [comment]
Just as an update, I fixed this via forcing the version to 7.1 in the install script. This version still works and should fix the script. Thanks again for letting me know.
